extensions/amp-animation/0.1/amp-animation.js

```diff
@@ -95,9 +95,11 @@ export class AmpAnimation extends AMP.BaseElement {
     const scriptElement = userAssert(
         childElementByTag(this.element, 'script'),
         '"<script type=application/json>" must be present');
-    this.configJson_ = tryParseJson(scriptElement.textContent, error => {
-      throw user().createError('failed to parse animation script', error);
-    });
+    this.configJson_ = /** @type {?JsonObject} */ (
+      tryParseJson(scriptElement.textContent, error => {
```

Comment:
Changed `tryParseJson` to return `null` in the failing case instead of `undefined`. With this, there's no need for the cast in this file. Done in https://github.com/ampproject/amphtml/pull/21618/commits/7cc19f024bf8e6e396b694144d0a6f8f1e5133b3.